### PR TITLE
feat: [pipe-23934]: fix pr list page - branch icon

### PR DESCRIPTION
**Pull request by cjlee01**
change the milestone icon to branch and make it relevant since we dont actually have milestone concept
image

### CODE REVIEW

File: packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx [hunk 1]
```diff
@@ -9,6 +10,9 @@ interface PullRequestItemDescriptionProps {
   author: string
   sourceBranch: string
   timestamp: string
+  targetBranch: string
+  spaceId?: string
+  repoId?: string
```
Comment (sans-harness) [inline packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx:15]: do we only need this for just navigation? Can we use a relative path and not have these props at all?

File: packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx [hunk 2]
```diff
@@ -38,11 +45,28 @@ export const PullRequestItemDescription: FC<PullRequestItemDescriptionProps> = (
...
+              <Icon name="branch" size={11} className="mr-1 text-tertiary-background" />
+              <Text size={1} className="p-0.5 hover:underline">
+                {targetBranch}
+              </Text>
+            </Link>
+          </Button>
+
+          <span>&larr;</span>
+          <Button variant="secondary" size="xs" asChild>
+            <Link to={`/${spaceId}/repos/${repoId}/code/${sourceBranch}`}>
```
Comment (sans-harness) [inline packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx:63]: could we do relative paths?

**Review by sans-harness — APPROVED** [review]

File: packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx (hunk 1 above)
Comment (cjlee01) [reply packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx]: done

File: packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx (hunk 2 above)
Comment (cjlee01) [reply packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx]: done

**Pull request merged**